R/brreg.R

```diff
@@ -25,15 +25,51 @@
 #' df <- get_entity(c(974760843, 971524960))
 #'
 get_entity <- function(entity, raw_response = FALSE) {
-  dl <- lapply(entity, function(x) get_entity_safe(x, raw_response))
-  if (raw_response) return(dl)
+  dl <- lapply(entity, function(x)
+    get_brreg_safe(x, type = 'enheter', raw_response))
+  if (raw_response) return(invisible(dl))
   do.call('rbind', dl)
 }
 
+#' Get roles
+#'
+#' Fetch information about entity roles from the
+#' _Central Coordinating Register for Legal Entities_
+#' (Enhetsregisteret) based on the organisation number.
+#'
+#' @details
+#' See the
+#' [API documentation](https://data.brreg.no/enhetsregisteret/api/docs/index.html)
+#' for further details (in Norwegian only).
+#'
```

Comment:
Add `@seealso` to `get_entity()`. It's not possible to get roles for an entity using a name search. If you do not have an organisation number, you should first do a call to `get_entity()` and then call `get_roles()` using the response from the first call.

Reply:
Done.